lib/api/wpapi.js

```diff
@@ -0,0 +1,77 @@
+var $ = require('zepto-browserify').$;
+
+var name = 'WP.CORE';
+
+var baseUrl = 'https://public-api.wordpress.com/wp/v2';
```

Comment:
`wp/v2` is definitely more important for right now, since that's what has been merged into core and what we need to focus on testing.  Fine with me to ship support for `wp/v2` first.